console/frontend/src/main/frontend/src/app/components/datatable/datatable.component.scss

```diff
@@ -0,0 +1,75 @@
+table {
+  margin-bottom: 10px;
+}
+
+.dt-options {
+  display: flex;
+
+  .dt-length,
+  .dt-search {
+    flex: 1;
+  }
+
+  .dt-search label {
+    float: right;
+  }
+
+  label {
+    font-weight: 400;
+  }
+
+  select,
+  input {
+    border: 1px solid #bdbfc2;
+    border-radius: 4px;
+    outline: none;
+  }
+}
+
+.dt-pagination {
+  display: flex;
+
+  .dt-pagination-info,
+  .dt-pagination-control {
+    flex: 1;
+  }
+
+  .dt-pagination-info {
+    align-content: center;
+  }
+
+  .dt-pagination-control .dt-control-wrapper {
+    float: right;
```

Comment:
Isn't there a flex way to get this to the right?
Maybe `justify-content: flex-end` or a `space-between`.

Reply:
I could do `display: flex; justify-content: end;` just for this css query but float in this case works better since it doesn't screw up the small space between the input and the label text